flathub.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
with these values you can just remove the flathub.json file

Reply:
Ah, so we don't build for i386 etc anymore?

Reply:
not for a few years